src/test/java/uk/gov/pay/api/it/rule/RedisDockerRule.java

```diff
@@ -1,5 +1,7 @@
 package uk.gov.pay.api.it.rule;
 
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.rules.TestRule;
 import org.junit.runner.Description;
 import org.junit.runners.model.Statement;
```

Comment:
is this mixing junit 4 and 5? not sure if that matters or not even if it is

Reply:
ah ok I see, I didn't know it was possible to use them interchangeably

Reply:
That's the beauty of using interfaces, rather than extending base/abstract classes 🙂